### PR TITLE
add steps to configure utilization scheduler in server

### PR DESCRIPTION
**Pull request by kelshmo**
Pending the outcome of IT-723, we may want to record this additional step here.

### CODE REVIEW

**Comment by karawoo** [comment]
Thank you!

File: vignettes/deploying-dccvalidator.Rmd [hunk 1]
```diff
@@ -22,8 +22,8 @@ please read the
 The app is deployed on the server through the following steps:
 
 1. ssh into the Shiny Pro server and navigate to `/home/kwoo/ShinyApps/dccvalidator-app`
-1. `git pull` changes from GitHub
-1. To ensure packages are up-to-date, open an R session and run `renv::restore()`
+2. `git pull` changes from GitHub
```
Comment (karawoo) [inline vignettes/deploying-dccvalidator.Rmd]: FYI markdown will automatically increment the numbers in an ordered list. This is how the [rendered page](https://sage-bionetworks.github.io/dccvalidator/articles/deploying-dccvalidator.html) appears, even though we are currently using `1., 1., 1.,`
Screen Shot 2019-11-05 at 11 55 34 AM

**Comment by kelshmo** [comment]
IT-723 remains open - will this be relevant when we move to `reticulate`?

**Comment by karawoo** [comment]
It won't be necessary, but also shouldn't cause any problems and might improve app performance slightly.

File: vignettes/deploying-dccvalidator.Rmd [hunk 2]
```diff
@@ -37,6 +37,14 @@ stand up a new version that is customized for a different community):
 1. If needed, make any changes to the application's behavior by editing the
    files, or check out a branch that contains your changes
 1. To ensure packages are up-to-date, open an R session and run `renv::restore()`
+1. The Sage Shiny Server administrator must configure the utilization scheduler to ensure each connection has its own R process. 
+```
+location /users {
+  location /yourusername/ShinyApps {
```
Comment (karawoo) [inline vignettes/deploying-dccvalidator.Rmd]:
```suggestion
  location /yourusername/yourappname {
```

This seems to be the invocation that actually works, judging by what I currently see in the shiny server config file

File: vignettes/deploying-dccvalidator.Rmd [hunk 3]
```diff
@@ -37,6 +37,14 @@ stand up a new version that is customized for a different community):
 1. If needed, make any changes to the application's behavior by editing the
    files, or check out a branch that contains your changes
 1. To ensure packages are up-to-date, open an R session and run `renv::restore()`
+1. The Sage Shiny Server administrator must configure the utilization scheduler to ensure each connection has its own R process.
```
Comment (karawoo) [inline vignettes/deploying-dccvalidator.Rmd]:
```suggestion
1. To ensure each connection has its own R process, the Sage Shiny Server administrator must configure the utilization scheduler. This is not necessary for the app to function, but might improve performance.
```

**Pull request merged**